### ISSUE TITLE
.editorconfig for typescript

### ISSUE THREAD

**Issue by esanzgar** [maksimr/vim-jsbeautify#143]
Two questions:

I use jsbeautify from typescript files. In my .vimrc, I have:
`autocmd FileType typescript vnoremap <buffer>  <c-f> :call RangeJsBeautify()<cr>`

How can I setup my .editorconfig for typescript? These rules are ignored for typescript files in my .editorconfig:
```
[**.ts]
charset = utf-8
indent_style = space
indent_size = 3
insert_final_newline = true
trim_trailing_whitespace = true
```

Thanks

**Comment by maksimr** [comment]
@esanzgar now supported only ['js', 'css', 'html', 'jsx', 'json'](https://github.com/maksimr/vim-jsbeautify/blob/master/plugin/beautifier.vim#L31) types.

Does js-beautify support typescript?

Thanks!

**Comment by esanzgar** [comment]
In general, js-beautify works pretty well with typescript. I found just a couple of corner cases were js-beautify doesn't format typescript correctly.

Renaming all `[**.ts]` to `[**.js]` doesn't work. Changing all the extension `.ts` to `.js` is not an option.

Thank you anyway.

**Comment by maksimr** [comment]
@esanzgar No problem,

I have checked [it for typescript](https://github.com/maksimr/vim-jsbeautify/commit/7c586568716263e27449d9b3f2475636bcd1f4dc) and it works for me

**Comment by esanzgar** [comment]
I have `JsBeautify` and v1.7.5 of js-beautify:
```

**Comment by esanzgar** [comment]
@maksimr You are right, it works for typescript files when the .editorconfig is explicitly set.

My issue was that the .editorconfig is found and parsed through the `editorconfig-vim` plugin. That was my particular scenario and it didn't work me.

I have resolved the issue with this simple if statement:
https://github.com/maksimr/vim-jsbeautify/compare/master...esanzgar:master

**Comment by maksimr** [comment]
@esanzgar Thanks
